client/jetbrains/src/main/java/com/sourcegraph/cody/completions/CodyCompletionsManager.java

```diff
@@ -167,6 +168,14 @@ private CompletableFuture<Void> triggerCompletionAsync(
             });
   }
 
+  public static InlineCompletionItem removeUndesiredCharacters(InlineCompletionItem item) {
+    String newInsertText = item.insertText.replaceAll("\u200b", ""); // no zero-width spaces, pls
```

Comment:
I also got an LSEP (`U+2028`) char in another completion. Not sure why this is happening, but I think we should also drop those.

```suggestion
    String newInsertText = item.insertText.replaceAll("[\u200b\u2028]", ""); // no zero-width spaces or line separator chars, pls
```

Reply:
I filled in a unit test for it as well 👌